### PR TITLE
fixed shigatyper permission bugs

### PR DESCRIPTION
**Pull request by erinyoung**
- [X] This comment contains a description of what is in the pull request.

- [X] Build your own docker image using a Dockerfile
  - [X] Includes the recommended LABELS
- [X] (Optional) Dockerfile is built with best practices and has been approved by a linter (such as https://hadolint.github.io/hadolint/)
- [X] Ensure tool is listed in Program_Licenses.md
- [X] Ensure a simple container-specific README.md exists in the same directory as the Dockerfile (i.e. spades/3.12.0/README.md)
- [X] Update GitHub actions workflow if needed
- [X] Any files required for building are located in the same directory as the Dockerfile (i.e. spades/3.12.0/my_spades_tests.sh)
- [X] Have successfully run the workflow "Test <program name> image" in your forked repository

### CODE REVIEW

File: .github/workflow-templates/readme-template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# <program> container
```
Comment (erinyoung) [inline .github/workflow-templates/readme-template.md:1]: oh yeah. Didn't mean to include this, but it can stay in.

**Comment by erinyoung** [comment]
What did I do? 

I changed how it was installed. Instead of via conda, it's now installed with python3 from setup.py.

Shigatyper doesn't have versions per say, but it does have tags and conda package tar.gz files, so I decided to use those.

I thought about using git clone to a specific commit, but didn't think that was necessary.

File: .github/workflow-templates/readme-template.md (hunk 1 above)
Comment (kapsakcj) [reply .github/workflow-templates/readme-template.md:1]: sounds good!

Comment (kapsakcj) [reply .github/workflow-templates/readme-template.md:1]: sounds good!

**Comment by kapsakcj** [comment]
testing locally now. Good to see the automated test pass ✔️

**Comment by kapsakcj** [comment]
Looks good! I was able to run the container as non-root user and successfully run `shigatyper`.

```
$ docker build -t erin/shigatyper:2.0.1 shigatyper/2.0.1/

[...]

Successfully built 74695190b0cc
Successfully tagged erin/shigatyper:2.0.1

$ docker run --rm=True -u $(id -u):$(id -g) -v $(pwd):/data -it erin/shigatyper:2.0.1

I have no name!@6d0b40f4bf29:/data$ shigatyper --R1 SRR7738178_1.fastq.gz --R2 SRR7738178_2.fastq.gz -n SRR7738178-new-shigatyper-docker
[CRITICAL::ShigaTyper] ShigaTyper v2.0.0
[M::mm_idx_gen::0.008*1.20] collected minimizers
[M::mm_idx_gen::0.014*1.60] sorted minimizers
[M::main::0.018*1.47] loaded/built the index for 95 target sequence(s)
[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 95
[M::mm_idx_stat::0.019*1.45] distinct minimizers: 21938 (99.78% are singletons); average occurrences: 1.002; average spacing: 5.442; total length: 119664
[M::main] Version: 2.24-r1122
[M::main] CMD: minimap2 -d /tmp/tmppx_hnd48/ShigellaRef5.mmi /usr/local/lib/python3.8/dist-packages/ShigaTyper-2.0.1-py3.8.egg/shigatyper/resources/ShigellaRef5.fasta
[M::main] Real time: 0.021 sec; CPU: 0.030 sec; Peak RSS: 0.006 GB
[WARNING] Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.
[M::main::0.006*1.14] loaded/built the index for 95 target sequence(s)
[M::mm_mapopt_update::0.006*1.14] mid_occ = 1000
[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 95
[M::mm_idx_stat::0.007*1.12] distinct minimizers: 21938 (99.78% are singletons); average occurrences: 1.002; average spacing: 5.442; total length: 119664
[M::worker_pipeline::2.423*1.95] mapped 206814 sequences
[M::worker_pipeline::3.697*2.25] mapped 207044 sequences
[M::worker_pipeline::5.006*2.51] mapped 206570 sequences
[M::worker_pipeline::6.054*2.71] mapped 206510 sequences
[M::worker_pipeline::6.597*2.59] mapped 173320 sequences
[M::main] Version: 2.24-r1122
[M::main] CMD: minimap2 -ax sr /tmp/tmppx_hnd48/ShigellaRef5.mmi /data/SRR7738178_1.fastq.gz /data/SRR7738178_2.fastq.gz
[M::main] Real time: 6.604 sec; CPU: 17.097 sec; Peak RSS: 0.360 GB
[mpileup] 1 samples in 1 input files
Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid
[warning] samtools mpileup option `g` is functional, but deprecated. Please switch to using bcftools mpileup in future.
[mpileup] 1 samples in 1 input files
[CRITICAL::ShigaTyper.run.report] ** SRR7738178 is predicted to be Shigella sonnei form II.**
[CRITICAL::ShigaTyper.run.report]
[CRITICAL::ShigaTyper.run.report] Writing hits to SRR7738178-hits.tsv
        Hit     Number of reads Length Covered  reference length        % covered       Number of variants      % accuracy
0       ipaH_c  672     779     780     99.9    3.0     99.6
1       EclacY  49      267     1254    21.3    1.0     99.6
2       cadA    529     2100    2143    98.0    22.0    99.0
3       Ss_methylase    307     1830    1833    99.8    0.0     100.0
[CRITICAL::ShigaTyper.run.report] Writing final serotype prediction to SRR7738178.tsv
sample  prediction      ipaB    notes
SRR7738178      Shigella sonnei form II -
[CRITICAL::ShigaTyper.run.report] Complete in  4.668 milliseconds.

```

**Comment by erinyoung** [comment]
Nice! Does this pass your review?

**Comment by kapsakcj** [comment]
Yep, deploying the new docker image now. Should be up on dockerhub and quay shortly.

feel free to delete the shigatyper branch if you're done with it.

**Pull request merged**